### PR TITLE
add test and docs for webdav

### PR DESCRIPTION
**Pull request by zhijian-pro**

### CODE REVIEW

File: docs/zh_cn/reference/command_reference.md [hunk 1]
```diff
@@ -382,6 +382,112 @@ juicefs gateway [command options] META-URL ADDRESS
 `--keep-etag`<br />
 保留对象上传时的 ETag (默认: false)
 
+### juicefs webdav
+
+#### 描述
+
+启动一个 webdav 服务。
```
Comment (davies) [inline docs/zh_cn/reference/command_reference.md:389]: WebDAV

File: pkg/fs/http_test.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package fs
```
Comment (davies) [inline pkg/fs/http_test.go:1]: license header

**Comment by davies** [comment]
Can we find a external compatible test suite for WebDAV ?

**Comment by zhijian-pro** [comment]
@davies  There should be. I will look for it again. If I find it, I will put it in as an integration test.

**Comment by zhijian-pro** [comment]
@davies  litmus is worth trying http://www.webdav.org/neon/litmus/

**Pull request merged**